src/libraries/System.Threading.Channels/src/System/Threading/Channels/Channel.netcoreapp.cs

```diff
@@ -0,0 +1,36 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+
+namespace System.Threading.Channels
+{
+    /// <summary>Provides static methods for creating channels.</summary>
+    public static partial class Channel
+    {
+        /// <summary>Creates an unbounded prioritized channel usable by any number of readers and writers concurrently.</summary>
+        /// <returns>The created channel.</returns>
+        /// <remarks>
```

Comment:
nit: one overload does not describe `T`, while the other does

```suggestion
        /// <summary>Creates an unbounded prioritized channel usable by any number of readers and writers concurrently.</summary>
        /// <typeparam name="T">Specifies the type of data in the channel.</typeparam>
        /// <returns>The created channel.</returns>
        /// <remarks>
```